internal/matrix/database/matrix_queries.go

```diff
@@ -1,28 +1,76 @@
 package database
 
+import "fmt"
+
 const (
-	createMatrix    = "create matrix"
-	deleteMatrix    = "delete matrix by uuid"
-	getMatrix       = "get matrix by uuid"
-	getCourseMatrix = "get course and matrix by uuid"
-	listMatrix      = "list matrices"
-	updateMatrix    = "update matrix by uuid"
-	addSubject      = "adds subject to matrix"
-	removeSubject   = "remove subject from matrix"
+	returningMatrixColumns = `matrices.uuid, matrices.code, matrices.name, matrices.description,
+		matrices.created_at, matrices.updated_at`
+	// CREATE.
```

Comment:
(non-blocker): can we remove these comments? I think it would make the code cleaner